### PR TITLE
Remove postcards for gold offer from /about/gold

### PR DESCRIPTION
**Pull request by powerlanguage**
👓  @Deimos 

gold-page

### CODE REVIEW

File: reddit_gold/public/static/css/gold.less [hunk 1]
```diff
@@ -415,88 +391,6 @@ section#about-gold {
...
-        .of {
-            font-size: 17px;
-            margin: 20px 0;
-        }
-
-        .gold-typo {
-            display: inline-block;
-            width: 170px;
-            height: 40px;
-            background: url(../gold/reddit-gold.png) no-repeat;
```
Comment (Deimos) [inline reddit_gold/public/static/css/gold.less]: I think you can delete this reddit-gold.png as well.

File: reddit_gold/public/static/css/gold.less [hunk 2]
```diff
@@ -415,88 +391,6 @@ section#about-gold {
...
-        }
-    }
-
-    .sticker {
-        position: absolute;
-        top: 174px;
-        left: 306px;
-        width: 85px;
-        height: 86px;
-        background: url(../gold/sticker.png) no-repeat;
```
Comment (Deimos) [inline reddit_gold/public/static/css/gold.less]: sticker.png should be able to be deleted too.

**Review by Deimos — CHANGES_REQUESTED** [review]
Looks good, just a couple more things that can be deleted too.

**Comment by Deimos** [comment]
:nail_care:

**Comment by powerlanguage** [comment]
💇

**Comment by Deimos** [comment]
:fish:

**Review by Deimos — APPROVED** [review]

**Pull request merged**